includes/admin/settings/class-settings-display.php

```diff
@@ -41,64 +41,30 @@ public function __construct() {
 		 * @return array
 		 */
 		public function get_settings() {
-			$settings        = array();
+			$settings        = [];
 			$current_section = give_get_current_setting_section();
 
 			switch ( $current_section ) {
 				case 'display-settings':
-					$settings = array(
+					$settings = [
 						// Section 1: Display
-						array(
+						[
 							'id'   => 'give_title_display_settings_1',
 							'type' => 'title',
-						),
-						array(
-							'name'    => __( 'Default GiveWP Styles', 'give' ),
-							'desc'    => __( 'You can disable Give\'s default styles for donation forms and other frontend elements.', 'give' ),
-							'id'      => 'css',
-							'type'    => 'radio_inline',
-							'default' => 'enabled',
-							'options' => array(
-								'enabled'  => __( 'Enabled', 'give' ),
-								'disabled' => __( 'Disabled', 'give' ),
-							),
-						),
-						array(
-							'name'    => __( 'Floating Labels', 'give' ),
```

Comment:
@henryholtgeerts `Floating Labels` setting removed without backward compatibility. I will suggest moving this setting in `Advanced -> Deprecated Settings`. Add a description to this section which will say that this section contains deprecated settings and will be moved in the future.

@DevinWalker Any thought on above suggestion

Reply:
I'd rather remove it entirely with backwards compatibility. Meaning, the global setting will no longer appear but if you have it enabled and update the existing legacy forms will still display floating labels.

Reply:
@DevinWalker got it. if someone wants to disable it then they can disable it from legacy form template setting.